terraform/README.md

```diff
@@ -0,0 +1,30 @@
+# Test environment Terraform code
+
+This terraform code creates a test environment for all invariants of `stack-mitosis`:
+
+- MySQL
+  - Same VPC
+  - Different VPC
+- Postgres
+  - Same VPC
+  - Different VPC
+
+## How to use
+
+```
+$ terraform apply
+```
+
+Wait some 15 minutes.
+
+See the output:
+
+```
+
+```
+
+Run stack-mitosis against all pairs of source and target instances:
+
+```
+
+```
```

Comment:
It looks like this is missing output? Maybe just leave the command sequence or a small subset of expected output? Prefer that to the empty space, but up to you for how to handle it.

Reply:
Oops, pushed a fix!